locations/spiders/darjeeling_fr.py

```
@@ -0,0 +1,39 @@
+
+    def post_process_item(self, item: Feature, response: TextResponse, feature: dict) -> Iterable[Feature]:
+        apply_category(Categories.SHOP_CLOTHES, item)
+        apply_clothes(Clothes.UNDERWEAR, item)
+
+        name = item.pop("name", "") or ""
+        item["branch"] = name.removeprefix("Darjeeling").removeprefix(" ")
+        try:
+            if hours := feature.get("hours"):
+                item["opening_hours"] = self.parse_hours(hours)
+        except Exception:
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)